webClient/src/app/shared/dialog/overwrite-dataset/overwrite-dataset.component.scss

```diff
@@ -0,0 +1,22 @@
+
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+  
+  SPDX-License-Identifier: EPL-2.0
+  
+  Copyright Contributors to the Zowe Project.
+*/
+mat-dialog-actions {

```

Comment:
These 3 lines are present in like 10 files, can you make a dialog.component.scss in
zlux-editor\webClient\src\app\shared\dialog\

that just has these 3 lines once & then every component that then uses its own SCSS file can just import this base on in CSS? Or put these 3 lines in zlux-editor\webClient\src\styles.scss (then just make sure these components once again, reference this styles SCSS)